### ISSUE TITLE
Run shell commands for command segment directly in shell

### ISSUE THREAD

**Issue by AjayKMehta** [JanDeDobbeleer/oh-my-posh#47]
In comments for #26, I mentioned wanting to create a segment that displays an icon only if `(Get-PSReadLineOption).HistorySaveStyle -ne 'SaveNothing'`. This does not work correctly because each time, the command gets run in a new instance of `pwsh`.

Theme used to test this: https://gist.github.com/AjayKMehta/1b5324a485a9db6e7d326276ab5786cf

@JanDeDobbeleer said he would look into seeing if he can modify the code so that a command can be run directly in your shell fixing this problem.

**Comment by JanDeDobbeleer** [comment]
@AjayKMehta unfortunately running direct commands isn't possible given the compiled nature of the beast. Although I think there might be another way to offer this functionality without generating a performance hit.

I'll try to draft a solution later today/tomorrow.

**Comment by JanDeDobbeleer** [comment]
@AjayKMehta I found a reasonable fix by thinking out of the box.
Will be available once I merge #49 

```powershell
Import-Module oh-my-posh
Set-PoshPrompt jandedobbeleer

function Set-EnvVar {
    $env:POSH=$(if ((Get-PSReadLineOption).HistorySaveStyle -ne 'SaveNothing'){'💾'} else {''})
}
New-Alias -Name 'Set-PoshContext' -Value 'Set-EnvVar' -Scope Global
```
<img width="964" alt="image" src="https://user-images.githubusercontent.com/2492783/95621028-56df7780-0a71-11eb-891d-830397de7c3a.png">
